lib/fields.c

```diff
@@ -66,7 +68,8 @@ int valid_field (const char *field, const char *illegal)
  * prompt the user with the name of the field being changed and the
  * current value.
  */
-void change_field (char *buf, size_t maxsize, const char *prompt)
+void
```

Comment:
Really, if you must do this, can't you just do a huge clang-format pass?

Reply:
Hmm, I'll have a look at using it.  Thanks for the suggestion.